modules/codegen/src/smithy4s/codegen/internals/SmithyToIR.scala

```diff
@@ -738,19 +738,6 @@ private[codegen] class SmithyToIR(model: Model, namespace: String) {
       Hint.PackedInputs
     case d: DeprecatedTrait =>
       Hint.Deprecated(d.getMessage.asScala, d.getSince.asScala)
-    case doc: DocumentationTrait =>
```

Comment:
Moved this out to a separate function to make it easier to get member docs even if the shape itself has no doc trait.